---
layout: post
title: "Third Generation Sequencing"
date: 2020-02-14
---

This is a blog post I wrote for my Written Professional Communication Course. It discusses briefly the history of genomic sequencing, the impact and problems of Illumina (Next-gen) Sequencing, and the potential methods and roadblocks of Third Gen Sequencing (3GS).

## History of Previous Sequencing Techniques
In 2019, if a person wants to have their entire genome sequenced, it will run them a small price of about $1000. This will allow anyone to see their unique combination of over 3 billion base pairs across their entire genome. However, around 10 years ago, this process cost over $100,000 and 10 years before that, it cost over $100 million.[1] How did we get to a point where virtually any individual can afford to have their DNA sequenced?

## Sanger Sequencing - The first glimpse of what makes us, us
The advancements in DNA sequencing technology have increased exponentially and we are seeing the cost and efficiency of genome sequencing decay at a similar rate. In the late 70s, Sanger developed the first efficient method of sequencing DNA using chain-termination. In short, Sanger divided the four base pairs and added a similar radiolabeled base to just one of the subjects. The other bases would replicate normally while the corresponding base was much shorter. Doing this for all of the bases and running the sample through gel electrophoresis separated the four samples and the radiolabeled nucleotides could be visualized on the gel. From the gel, one could see with their eye the sequence of that DNA fragment.
Next-Generation Sequencing
This process continued to be the de-facto method if your experiment required extensive DNA sequencing for the next forty years. Around the middle of the 2000s, a new sequencing method was developed by the biotechnology company Illumina. Illumina sequencing, or next-generation-sequencing (NGS), utilizes a massively parallel method to sequencing entire genomes in the fraction of the time it took for Sanger sequencing. The simplified method for this process is to divide the genome into small fragments, randomly sampling for a fragment, and sequencing it using one of a variety of technologies depending on what you want your output to look like. For example, you can place your DNA on a small glass sheet and label the different bases. Then, using a camera and a computer, you can track which bases sequence when and develop a sequence from the colored dots based on which dots appeared in order. 

Figure 1 – How Illumina sequencing photos appear on computer software. Note the different colors of each base pair:
![alt text]("https://github.com/kev08rac/kev08rac.github.io/blob/master/blog/images/illumina.png")

In my opinion, the impact of Illumina sequencing on the field of genomics is on the same scale as the impact of the smartphone on the mobile phone market. Being able to sequence entire genomes quickly has created the field of personalized medicine. For example, if a couple became pregnant, the mother could get her child’s embryo sequenced and be able to see if their child would have any genetic diseases such as down syndrome. In addition to human benefits, we can also use this technique to sequence other organism’s genomes. This greatly speeds up important research being done on those organisms that will be beneficial to help us understand them better.
While there are many upsides to NGS, other genetic techniques have brought an ethical debate into DNA sequencing, especially on humans. As I mentioned earlier, DNA sequencing a child before they’re born can allow us to determine if they will be born with any birth defects or genetic diseases. We can then take this information and use gene therapy to cure the disease. Gene therapy actually means going into the bad genome and using techniques, such as CRISPR, we can edit the genome and cure the disease. While this practice is not currently legal on humans, it has been proven to work on other organisms such as mice. There has been discussion that, if we determined which genes are linked to certain phenotypes, we can edit any part of the body we want for a child.
This brings into question whether or not doing this to create “perfect” humans is ethical and we are not playing God. For example, just a year ago a scientist in China [claimed to have used CRISPR to alter the genome of a pair of twins](https://www.livescience.com/64166-first-genetically-modified-babies-risks.html). He claimed that he successfully and safely removed the unwanted genes from the twins, and they were born successfully without any repercussions. As the article states, these claims have not been confirmed by the hospital, and the scientist seemed to have little regard for the repercussions of this experiment. After World War II, the [Declaration of Helsinki](https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/) is really the only piece of legislation we have for gene editing on humans. This document was formed after the horrific experiments carried out by the Nazi’s during the war. In the US, there is no specific legislation against this. However, in order to undergo such an experiment, you must first get it approved by the FDA for funding and approval for running trials on humans.
 
Figure 2 – A simplified flow chart of the CRISPR method of gene editing:
![alt text]("https://github.com/kev08rac/kev08rac.github.io/blob/master/blog/images/crispr.jpg")

While the technology and cost of gene editing seems to be right around the corner, the ethical debate on this topic will likely take many years to finally figure out. Such questions still need to be answered. Do we draw the line at any genetic illness or just life-threatening ones? Will an underground “black market” gene-editing industry open that can allow scientists to get rich by editing any gene the customer wants on their child?
We as a species have made incredible strides in the field of genomics in the last 40 years. We are able to look within ourselves and look at exactly what determines how tall we are, what color our eyes are, and what determines our hair color. And while we have made significant advancements, there are still new and exciting methods currently being developed to sequence DNA at an even faster and cheaper rate.

## New Wave of DNA Sequencing - Third-Generation Sequencing
Since the human genome project, the development of newer and better DNA sequencing technologies has led to the cost of sequencing genomes decreasing more than a thousand-fold.2 With the majority of biotech companies switching from Sanger or first-generation sequencing to NGS, the next generation of genome sequencing has already started being developed. In the broad sense, third-generation sequencing (3GS) works by reading the nucleotide sequences at the single-molecule level, in contrast to NGS that require breaking the long strands of DNA into smaller segments then inferring nucleotide sequences by amplification and synthesis, such as Illumina sequencing.
Potential Breakthrough Technologies for 3GS
There are a few different technologies currently being developed that attempt to tackle 3GS. One of these new technologies was developed by Pacific Biosciences called Single-Molecule Sequencing in Real-Time (SMRT). This system involves a single-stranded molecule of DNA that is attached to DNA polymerase (the enzyme that facilitates DNA transcription), which adds fluorescently labeled nucleotides to the DNA. As each leveled base is added, the fluorescent color of the base is recorded before the label is cut off.[2] The benefit of these technologies is that we can sequence very long reads which make it much easier to assemble them together versus many shorter reads.
 
Figure 3 – Line graph representing the speed of sequencing in kilo-bases per day:
![alt text]("https://github.com/kev08rac/kev08rac.github.io/blob/master/blog/images/dna_speed.png")

While the advancements of (3GS) seem very promising, it is still being developed and there are some major roadblocks we are running into, such as high error rates and cost. The effect of the error rates causes the sequencing software to become overwhelmed by error correction tasks.[4] To contextualize this, the whole assembly of a human genome using NGS Illumina sequencing data took about 24 CPU hours, 3GS data reported to have taken half a million CPU hours.[3] A CPU hour is one hour’s worth of 100% of the CPU’s processing power going towards that specific task. Obviously, we can parallelize this processing power by splitting up tasks and allocating them to different CPUs. However, half a million CPU tasks would still take a very long time to process even if split into many CPUs. The fiscal cost of 3GS sequencing is, albeit decreasing, still at least an order of magnitude more expensive than Illumina NGS sequencing.5

## Computational Roadblocks
In addition to advancements in the methods of 3GS, the software used to analyze this data also must be upgraded and optimized. The biggest difference between software used for NGS compared to 3GS is the increased assembly length of DNA. As I mentioned earlier, for NGS data, the DNA is broken into small fragments and transformed together later. For 3GS, these fragments of DNA are much longer. While this simplifies the physical assembly graph, this has a critical impact on the computational complexities of analysis.[5]
 
Figure 4 – A visual representation of the longest-path algorithm to find the consensus sequence:
![alt text]("https://github.com/kev08rac/kev08rac.github.io/blob/master/blog/images/graph.png")

From the image, we can see multiple runs of 3GS sequencing producing varying versions of this sequence. We have a consensus (backbone) sequence and 2 runs with different results, being a result of the lower accuracy 3GS offers. In order to build this consensus sequence, a max-path algorithm is conducted to check each sequence and build the one you want. In computer science, this algorithm goes to each node in the graph and checks each node connecting to it and then moves to the next node and repeats. In the end, it will calculate the path it took to get to those nodes. While this graphing strategy was effective for old-school Sanger sequencing, the number of short reads generated through NGS caused too much computational overhead as linking the fragments was way too inefficient. 3GS data makes this approach more feasible, but the high error rates render this method currently unusable.[7]
One solution some researches are using right now is combining the techniques of NGS and 3GS to implement a hybrid approach of sequencing. [In this article](https://www.nature.com/articles/srep31900), the authors used 3GS sequencing techniques on NGS data. This lowers the computational burden of incorrect sequencing reads for 3GS sequencing techniques.

## What Could the Future Hold?
3GS has the potential to sequence DNA much faster than Illumina NGS. The computational benefits alone allow for much more efficient DNA analysis. There are an estimated 8.7 million species of life on Earth, with only 1.3 million discovered. 3GS sequencing can propel us in sequencing and mapping every discovered species. We can study species-specific diseases and physiological interactions between different species. One especially exciting method is to determine which genes code for beneficial proteins in other species that we can use to create medicines or other helpful compounds. The monetary benefits of 3GS can allow the already fairly cheap cost of sequencing one’s own DNA to be even cheaper. Cheaper costs can also help researchers financially, as they would be able to ask for less money on their grants or allocate the extra money to another part of their research experiment for better analysis.
In conclusion, we have accelerated greatly in the processes of sequencing DNA since the 1970s and Sanger sequencing. I am very excited to be able to experience the rise of 3GS and hope it will have the same impact on NGS that NGS had on Sanger sequencing. I anticipate this will continue to increase the socioeconomic demographic eligible for personalized medicine, as well as advancements in gene therapy.

## References
[1]: Van Dijk EL, Jaszczyszyn Y, Naquin D, Thermes C. The Third Revolution in Sequencing Technology. Trends in genetics: TIG. 2018 Sep [accessed 2019 Nov 13]. https://www.ncbi.nlm.nih.gov/pubmed/29941292
[2]: Third Generation Sequencing. 2016 Jun 13. YourGenome; [accessed 2019 Nov 13]. https://www.yourgenome.org/stories/third-generation-sequencing
[3,4,5,6,7]: Ye C, Hill CM, Wu S, Ruan J, Ma Z(S. DBG2OLC: Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation Sequencing Technologies. 2016 Aug 30 [accessed 2019 Nov 13]. https://www.nature.com/articles/srep31900

